mobile-menu.js

```diff
@@ -0,0 +1,26 @@
+const toggleButton = document.querySelector('.toggle-button')
+const menuButton = document.querySelector('.x-button')
+const navbarLinks = document.querySelector('.nav-ul')
+const bodyElement = document.querySelector('body')
+const featuredSpeaker = document.querySelector('.featured-speakers-ul')
+
+toggleButton.addEventListener('click', () => {
+  navbarLinks.classList.toggle('display-none')
+  toggleButton.classList.add('display-none')
+  menuButton.classList.toggle('display-none')
+  bodyElement.style.overflow = 'hidden'
+})
+
+menuButton.addEventListener('click', () => {
+  navbarLinks.classList.toggle('display-none')
+  toggleButton.classList.toggle('display-none')
+  menuButton.classList.toggle('display-none')
+  bodyElement.style.overflow = 'auto'
+})

```

Comment:
- [ ] Please make sure to fix the menu button when it's changing from the mobile menu to the desktop, it still appears on the page.

Reply:
Thanks for the review change made